_inc/client/setup-wizard/feature-toggle/map-feature-to-props.js

```diff
@@ -0,0 +1,1028 @@
+/**
+ * External dependencies
+ */
+import { translate as __ } from 'i18n-calypso';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import getRedirectUrl from 'lib/jp-redirect';
+import { getPlanClass } from 'lib/plans/constants';
+import { getVaultPressData, isAkismetKeyValid } from 'state/at-a-glance';
+import { getSiteRawUrl } from 'state/initial-state';
+import { getRewindStatus } from 'state/rewind';
+import { getSetting, updateSettings } from 'state/settings';
+import { getSitePlan, hasActiveSearchPurchase } from 'state/site';
+
+const features = {
+	ads: {
+		mapStateToProps: state => {
+			const sitePlan = getSitePlan( state );
+			const planClass = getPlanClass( sitePlan.product_slug );
+
+			const inCurrentPlan = [ 'is-premium-plan', 'is-business-plan' ].includes( planClass );
+
+			let upgradeLink;
+			if ( ! inCurrentPlan ) {
+				upgradeLink = '#/plans';
+			}
+
+			let info;
+			let configureLink;
+			if ( inCurrentPlan ) {
+				info = __( `Included with ${ sitePlan.product_name }` );
```

Comment:
I don't think this will work. I would recommend doing something like this instead:

```suggestion
				info = __(
					'Included with %(product_name)s', {
						args: {
							product_name: sitePlan.product_name,
						},
						comment: '%(product_name) is the name of a product you can purchase.',
					}
				);
```